exercise/cache/points.py

```diff
@@ -356,14 +356,14 @@ def check_reveal_rule() -> None:
                             'best_submission': submission.id,
                             'points': submission.grade,
                             'formatted_points': format_points(submission.grade, True, False),
-                            'passed': (ready and submission.grade >= entry['points_to_pass']),
-                            'graded': ready, # != unofficial
+                            'passed': (not unofficial and submission.grade >= entry['points_to_pass']),
+                            'graded': not unofficial,
```

Comment:
If all submissions are in the INITIALIZED status, then it is strange to call the exercise "graded" since the submissions really have not been graded yet.

Reply:
So do you think the definition of "graded" should remain unchanged (best submission's status == READY)? In that case, I'll make the same change to UserExerciseSummary as well.

What about "passed"? Do you think it's strange to call an exercise "passed" if all of its submissions are INITIALIZED?